content/docs/studio/get-started.md

```diff
@@ -0,0 +1,64 @@
+# Get Started with DVC Studio
+
+Here, we will walk you through a tutorial to use DVC Studio for collaboration on
+your ML projects. You will need access to a Github, Gitlab or Bitbucket account
+which has access to the Git repositories you want to connect to from DVC Studio.
+DVC Studio creates views from repositories when you connect to them.
+
+## What is a view?
+
+A _view_ is an interactive representation of the experiments run in your ML
+project. DVC Studio automatically identifies datasets, metrics and
+hyperparameters in your ML experiments. You can also add custom files with the
+required data (more on this later). Using this data, DVC Studio creates a
+summary view. This includes tables of all the metrics across all your
+experiments. You can also generate plots and compare experiments here.
+
+## Access your first view
+
+In your browser, open <https://viewer.iterative.ai>. Sign in with your Github,
+Gitlab, or Bitbucket account.
+
+![](/img/studio/login_home.png) _When you first login, an example view is
+already created for you to explore, and you can add more views._
+
+When you first login, you will find that there already exists a view connecting
+to an example DVC project. Use this view to explore the features that DVC Studio
+has to offer.
```

Comment:
> When you first login, you will find that there already exists a view connecting to an example DVC project.

This did not happen for me. I just see:

![image](https://user-images.githubusercontent.com/1477535/117891601-cd995900-b27c-11eb-9f21-86c4024025e8.png)

Reply:
> Also the paragraph is repetitive with the img caption. I'd change the caption to _DVC Studio Views page_

Reply:
> This did not happen for me.
A separate task is ongoing to add the example view by default.

> change the caption to just DVC Studio Views page 
sounds good. will change.

Reply:
> A separate task is ongoing to add the example view by default.

Meaning a docs issue? Didn't find this in #2468 or #2469

Reply:
there's a demo view that loads by default for new users now.

Reply:
Great, thanks for the follow-up!